src/applications/ask-va/config/schema-helpers/personalInformationHelper.js

```diff
@@ -182,155 +200,190 @@ export const personalInformationUiSchemas = {
     'ui:title': ssnServiceInfo,
     'ui:required': () => true,
     'ui:validations': [validateGroup],
-    'ui:options': {
-      showFieldLabel: true,
-    },
-    ssn: merge(ssnUI, { 'ui:options': { widgetClassNames: null } }),
-    serviceNumber: {
-      'ui:title': 'Service number',
-      'ui:errorMessages': {
-        pattern:
-          'Veteran service number must start with 0, 1, or 2 letters followed by 5 to 8 digits',
-      },
-      'ui:options': {
-        replaceSchema: () => {
-          return {
-            type: 'string',
-            pattern: '^[a-zA-Z]{0,2}\\d{5,8}$',
-            maxLength: 8,
-          };
-        },
-      },
-    },
-  },
-  dob: {
-    ...currentOrPastDateUI('Date of birth'),
-    'ui:required': () => true,
+    'ui:options': { showFieldLabel: true },
+    ssn: ssnUI(),
+    serviceNumber: serviceNumberUI('Service number'),
   },
-  pronouns: {
-    'ui:title': pronounInfo,
-    'ui:required': () => true,
-    'ui:validations': [validateGroup],
+  socialNum: {
+    ...ssnUI(),
+    'ui:required': () => false,
     'ui:options': {
-      showFieldLabel: true,
-    },
-    ...createUiTitlePropertiesFromOptions(pronounsLabels),
-    pronounsNotListedText: {
-      'ui:title':
-        'If not listed, please provide your preferred pronouns (255 characters maximum)',
+      hideIf: () => true,
     },
   },
-  genderIdentity: {
-    'ui:title': 'Gender identity',
-    'ui:description': genderInfo,
-    genderIdentity: {
-      'ui:widget': 'radio',
-      'ui:title': `Select a gender identity`,
-      'ui:required': () => true,
-      'ui:options': {
-        labels: genderLabels,
-        enumOptions: genderOptions,
-      },
+  branchOfService: {
+    'ui:title': 'Branch of service',
+    'ui:webComponentField': VaSelectField,
+    'ui:options': {
+      hideIf: () => true,
     },
   },
+  dateOfBirth: { ...dateOfBirthUI(), 'ui:required': () => true },
 };
+// dob: { ...currentOrPastDateUI('Date of birth'), 'ui:required': () => true },
```

Comment:
Please note pronouns, and gender are commented out as they will be revisited later pending a vote.